### PR TITLE
Add selected FAQ questions

### PR DESCRIPTION
**Pull request by cari-lynn**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Sometimes Kubernetes API server will convert submitted field values into their c
 ...
 ```
 
-Consider using [ytt](https://get-ytt.io) and [its overlay feature](https://github.com/k14s/ytt/blob/develop/docs/lang-ref-ytt-overlay.md) to change values if you do not control source configuration.
+Consider using [ytt](https://get-ytt.io) and [its overlay feature](lang-ref-ytt-overlay.md) to change values if you do not control source configuration.
```
Comment (cppforlife) [inline docs/faq.md]: wrong repo.

File: docs/faq.md [hunk 2]
```diff
@@ -71,3 +71,60 @@ There might be cases where other system actors (various controllers) may modify
...
+
+Resources: [merge method](merge-method.md), [rebase rules](config.md#rebaserules)
+
+---
+#### Can I force an update for a change that does not produce a diff?
+
+If kapp does not detect changes, it won't perform an update. To force changes regardless you can provide the [`kapp.k14s.io/nonce`](apply.md#kappk14siononce) annotation. That way, every time you deploy the resource will appear to have changes.
+
+---
+#### How can I use kapp with AWS and terraform provider?
```
Comment (cppforlife) [inline docs/faq.md]: this question is pretty confusing. i think this should be removed.

File: docs/faq.md [hunk 3]
```diff
@@ -71,3 +71,60 @@ There might be cases where other system actors (various controllers) may modify
...
+Be sure to replace host, ca, token and namespace fields with your values.
+---
+#### How can I push an app to a namespace that may or may not already exist?
+
+kapp cannot to deploy to a nonexistent namespace since kapp needs to save generated label (in a configmap) before creating app resources. You can target a namespace with `-n.`
+
+Resources: [State Namespace](state-namespace.md)
+
+---
+#### How can I get the raw kapp inspect diff?
```
Comment (cppforlife) [inline docs/faq.md]: "How can I get the raw kapp inspect diff?" -> "How can I remove decorative headings from kapp inspect output?"

File: docs/faq.md [hunk 4]
```diff
@@ -71,3 +71,60 @@ There might be cases where other system actors (various controllers) may modify
...
+#### Can I force an update for a change that does not produce a diff?
+
+If kapp does not detect changes, it won't perform an update. To force changes regardless you can provide the [`kapp.k14s.io/nonce`](apply.md#kappk14siononce) annotation. That way, every time you deploy the resource will appear to have changes.
+
+---
+#### How can I use kapp with AWS and terraform provider?
+
+You can use a kubeconfig like [this](https://github.com/k14s/kapp-controller/blob/develop/pkg/deploy/service_accounts.go#L77)
+Be sure to replace host, ca, token and namespace fields with your values.
+---
```
Comment (cppforlife) [inline docs/faq.md]: newline before triple-dash

File: docs/faq.md (hunk 2 above)
Comment (cari-lynn) [reply docs/faq.md]: You mean for ` Can I force an update for a change that does not produce a diff?`

**Pull request merged**